src/btree/row_modify.c

```diff
@@ -317,16 +317,18 @@ __wt_update_obsolete_check(
     WT_TXN_GLOBAL *txn_global;
     WT_UPDATE *first, *next;
     size_t size;
-    uint64_t oldest, stable;
+    uint64_t oldest, stable, min_ts;
```

Comment:
Styling: variable names should be in alphabetical order.

Reply:
done!